### ISSUE TITLE
SPDY not working in 0.8 RC2 for Chrome

### ISSUE THREAD

**Issue by derekcicerone** [dropwizard/dropwizard#856]
I just updated to DropWizard 0.8 RC2 with a spdy3 connector. SPDY works fine from Firefox but doesn't seem to work for Chrome. I debugged this a bit and the issue may be in Spdy3ConnectorFactory. It is setting up ALPN with http/1.1 and spdy3 protocols which is used in this code in ALPNServerConnection:
```
    public String select(List<String> clientProtocols)
    {
        List<String> serverProtocols = getProtocols();
        String negotiated = null;
        for (String clientProtocol : clientProtocols)
        {
            if (serverProtocols.contains(clientProtocol))
            {
                negotiated = clientProtocol;
                break;
            }
        }
```

When testing with the latest Chrome release, it looks like the "clientProtocols" is coming in as [http/1.1, spdy/3, spdy/3.1]. This seems to result in http/1.1 being selected instead of SPDY.

**Comment by derekcicerone** [comment]
I believe this is the fix:
https://eclipse.googlesource.com/jetty/org.eclipse.jetty.project/+/f02da0cc339ace0a663c6bd16bfd3012f1f94879%5E%21/#F0

**Comment by joschi** [comment]
Fixed by upgrading to Jetty 9.2.9.v20150224 in 071b8104c0cf788264594899386ea5bfbf28d910.

eclipse/jetty.project@f02da0cc339ace0a663c6bd16bfd3012f1f94879
